examples/immutable-values/immutable_values.bal

```diff
@@ -61,7 +61,7 @@ public function main() {
     if (updateResult is error) {
         // An error should occur since `m2` is immutable.
         io:println("Error occurred on update: ",
-                   <string>updateResult.detail()["message"]);
+                   updateResult.detail()["message"].toString());
```

Comment:
The issue here is in the cast, because now, if the source type includes error the target type should also include error.

Shall we just do
```suggestion
                   updateResult.detail()["message"]);
```